src/catalog/src/builtin.rs

```diff
@@ -3314,6 +3314,30 @@ pub static MZ_COMMENTS: LazyLock<BuiltinTable> = LazyLock::new(|| BuiltinTable {
     access: vec![PUBLIC_SELECT],
 });
 
+pub static MZ_SOURCE_REFERENCES: LazyLock<BuiltinTable> = LazyLock::new(|| BuiltinTable {
+    name: "mz_source_references",
+    schema: MZ_INTERNAL_SCHEMA,
+    oid: oid::TABLE_MZ_SOURCE_REFERENCES_OID,
+    desc: RelationDesc::empty()
+        .with_column("source_id", ScalarType::String.nullable(false))
+        .with_column("namespace", ScalarType::String.nullable(true))
+        .with_column("name", ScalarType::String.nullable(false))
+        .with_column(
+            "updated_at",
+            ScalarType::TimestampTz { precision: None }.nullable(false),
+        )
+        .with_column(
+            "columns",
+            ScalarType::List {
+                element_type: Box::new(ScalarType::String),
+                custom_id: None,
+            }
+            .nullable(true),
```

Comment:
This probably shouldn't actually be null, right? If there are no columns it should be `[]` not `NULL`.

Reply:
Ah brilliant, thanks! Sorry for the onise.